generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -3384,9 +3385,13 @@ def write_to_teds_from_file(
                 basic_teds_options_raw=basic_teds_options))
 
 def _assign_numpy_array(numpy_array, grpc_array):
-    """ Assigns the grpc array to the numpy array, while still maintaining the 
-    original shape of the numpy array. 
+    """ 
+    Assigns the grpc array to the numpy array, while still maintaining the 
+    original shape of the numpy array.
+    Checks if grpc_array is an instance of bytes, then converts grpc_array to a 1D array. 
     """
```

Comment:
https://peps.python.org/pep-0257/#multi-line-docstrings

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.

Reply:
Restructured the multiline docstring.

Reply:
There should be a one line summary, then a blank line, then additional paragraphs separated by blank lines.

Reply:
Thank you for the example snippet. I have updated the docstring to this format.